content/20.treatments.md

```diff
@@ -940,8 +940,8 @@ Upcoming work may explore the specificity of nAbs for their target, mechanisms b
 There is also some research into possible potential therapeutics or prophylactics which interact with components of the innate immune response.
 For example, there are a variety of toll-like receptors (TLRs) which are examples of pattern recognition receptors (PRRs), innate immune response components which recognize pathogen-associated molecular patterns (PAMPs) and damage-associated molecular patterns (DAMPs).
 Note that TLRs form a part of innate immune recognition and can more generally contribute to promoting both innate and adaptive responses [@ISBN:9780815332183].
-In mouse models, poly(I:C) and lipopolysaccharide, which are agonists of toll-like receptors TLR3 and TLR4, respectively, showed protective effects when administered prior to SARS-CoV exposure [@doi:10.1128/JVI.01410-12].
```

Comment:
change word exposure to infection (more biologically precise). i.e., should be for clarity and specificity to reference: "...prior to SARS-CoV infection [..."
again, this sentence and the next were part of the draft of #349 for edits (the 2nd file) and we decided later to combine it with other similar ideas into #408. It was a great article find, but since it was just a draft requires a bit of fixing so that that it corresponds better to the reference. (Also if we keep "antagonists" in next sentence without explaining then it might superficially sound like either we are not using the prophylaxis in the reference or that we are precluding effective use of the prophylaxis in the reference, unless it's part of a more complex strategy which needs to be described; in order to explain why "antagonist" might be something worth researching we'd need more references and description.)